lib/zold/node/front.rb

```diff
@@ -94,7 +94,13 @@ class Front < Sinatra::Base
     end
 
     get '/favicon.ico' do
-      redirect 'https://www.zold.io/logo.png'
+      if score >= 16
```

Comment:
@oltdaniel are you sure the test passes? This line should not work, since `score` is of type `Score` and it can't be compare with `Integer`

Reply:
Test passed without an error.

Reply:
@oltdaniel I can't understand how it's possible :) Anyway, let's merge and deploy and see how production will behave.